src/renderer/components/query-result-table.jsx

```diff
@@ -336,13 +336,11 @@ export default class QueryResultTable extends Component {
    * It gives a better UX since the column adapts better to the table width.
    */
   resolveCellWidth(fieldName, fields, rows) {
-    const { customFont } = this.props.config.data.customFont;
-    const font = `14px '${customFont}', 'Helvetica Neue', Arial, Helvetica, sans-serif`;
+    const customFont = this.props.config.data.customFont || 'Helvetica Neue';
+    const font = `14px '${customFont}' 'Helvetica Neue', Arial, Helvetica, sans-serif`;
```

Comment:
You removed the `,` from the customFont. I think the comma is required there.

Reply:
yes you are correct. I will fix that.